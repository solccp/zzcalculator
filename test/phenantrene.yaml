---
structure_id: 1
number_of_atoms: 14
double_bonds:
- [2, 3]
- [1, 12]
- [14, 13]
- [11, 6]
- [4, 7]
- [5, 10]
- [8, 9]
---
structure_id: 2
number_of_atoms: 14
double_bonds:
- [2, 3]
- [1, 12]
- [14, 13]
- [11, 6]
- [4, 5]
- [7, 8]
- [9, 10]
---
structure_id: 3
number_of_atoms: 14
double_bonds:
- [2, 3]
- [1, 12]
- [14, 13]
- [11, 6]
rings:
- [4, 5, 10, 9, 8, 7]
---
structure_id: 4
number_of_atoms: 14
double_bonds:
- [2, 3]
- [1, 6]
- [11, 13]
- [12, 14]
- [4, 7]
- [5, 10]
- [8, 9]
---
structure_id: 5
number_of_atoms: 14
double_bonds:
- [2, 3]
- [1, 6]
- [11, 13]
- [12, 14]
- [4, 5]
- [7, 8]
- [9, 10]
---
structure_id: 6
number_of_atoms: 14
double_bonds:
- [2, 3]
- [1, 6]
- [11, 13]
- [12, 14]
rings:
- [4, 5, 10, 9, 8, 7]
---
structure_id: 7
number_of_atoms: 14
double_bonds:
- [2, 3]
- [4, 7]
- [5, 10]
- [8, 9]
rings:
- [1, 6, 11, 13, 14, 12]
---
structure_id: 8
number_of_atoms: 14
double_bonds:
- [2, 3]
- [4, 5]
- [7, 8]
- [9, 10]
rings:
- [1, 6, 11, 13, 14, 12]
---
structure_id: 9
number_of_atoms: 14
double_bonds:
- [2, 3]
rings:
- [1, 6, 11, 13, 14, 12]
- [4, 5, 10, 9, 8, 7]
---
structure_id: 10
number_of_atoms: 14
double_bonds:
- [1, 2]
- [3, 4]
- [7, 8]
- [9, 10]
- [5, 6]
- [11, 13]
- [12, 14]
---
structure_id: 11
number_of_atoms: 14
double_bonds:
- [7, 8]
- [9, 10]
- [11, 13]
- [12, 14]
rings:
- [1, 2, 3, 4, 5, 6]
