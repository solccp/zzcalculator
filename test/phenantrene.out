 5 + 5 x + 1 x^2
 total: 11
